Building on Fedora30 fails

enki and aseba cmakes tell me that it cannot find Boost
```
-- The following packages have not been found:

 * Boost (required version >= 1.67)

```
But boost is installed:
```
boost-devel.x86_64                                1.69.0-6.fc30                             @fedora                           
boost-python2-devel.x86_64                        1.69.0-6.fc30                             @fedora                           
boost-python3-devel.x86_64                        1.69.0-6.fc30                             @fedora     
```
enki builds though.

When I try to build aseba, it errors out with this error:
```
[ 42%] Automatic MOC for target asebastudio
[ 42%] Built target asebastudio_autogen
[ 42%] Building CXX object aseba/clients/studio/CMakeFiles/asebastudio.dir/VariablesModel.cpp.o
/home/nik/src/aseba/aseba/aseba/clients/studio/VariablesModel.cpp: In lambda function:
/home/nik/src/aseba/aseba/aseba/clients/studio/VariablesModel.cpp:75:76: error: lambda parameter ‘key’ previously declared as a capture
   75 |                                    [&key](const auto& ptr, const QVariant& key) { return lessThan(ptr->key, key); });
      |                                                            ~~~~~~~~~~~~~~~~^~~
make[2]: *** [aseba/clients/studio/CMakeFiles/asebastudio.dir/build.make:284: aseba/clients/studio/CMakeFiles/asebastudio.dir/VariablesModel.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:10985: aseba/clients/studio/CMakeFiles/asebastudio.dir/all] Error 2

```

Still errors out:
```
[ 46%] Building CXX object aseba/clients/studio/CMakeFiles/asebastudio.dir/VariablesModel.cpp.o
/home/nik/src/aseba/aseba/clients/studio/VariablesModel.cpp: In lambda function:
/home/nik/src/aseba/aseba/clients/studio/VariablesModel.cpp:75:76: error: lambda parameter ‘key’ previously declared as a capture
   75 |                                    [&key](const auto& ptr, const QVariant& key) { return lessThan(ptr->key, key); });
      |                                                            ~~~~~~~~~~~~~~~~^~~
make[2]: *** [aseba/clients/studio/CMakeFiles/asebastudio.dir/build.make:284: aseba/clients/studio/CMakeFiles/asebastudio.dir/VariablesModel.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:10985: aseba/clients/studio/CMakeFiles/asebastudio.dir/all] Error 2
make: *** [Makefile:163: all] Error 2

```

--- a/aseba/clients/studio/VariablesModel.cpp
+++ b/aseba/clients/studio/VariablesModel.cpp
@@ -72,7 +72,7 @@ namespace {
 
     VariablesModel::TreeItem* find_or_create_child(VariablesModel::TreeItem& item, const QVariant& key) {
         auto it = std::lower_bound(item.children.begin(), item.children.end(), key,
-                                   [&key](const auto& ptr, const QVariant& key) { return lessThan(ptr->key, key); });
+                                   [](const auto& ptr, const QVariant& key) { return lessThan(ptr->key, key); });
         if(it != item.children.end() && (*it)->key == key)
             return it->get();
 

